model/execution/DeliveryExecutionList.php

```diff
@@ -102,6 +102,9 @@ private function createExecution($cachedData, $extraFields)
             'allowExtraTime' => isset($cachedData[DeliveryMonitoringService::ALLOW_EXTRA_TIME])
                 ? (bool)$cachedData[DeliveryMonitoringService::ALLOW_EXTRA_TIME]
                 : null,
+            'allowTimerAdjustment' => $this->getDeliveryExecutionManagerService()->isTimerAdjustmentAllowed(
```

Comment:
With this change many tests in `DeliveryExecutionListTest` fail. We need to mock `DeliveryExecutionManagerService` service in ServiceLocator mock there.

Reply:
I've fixed the `DeliveryExecutionListTest` unit test.